### PR TITLE
Add k-Fold CV & wandb 

### PR DESCRIPTION
**Pull request by Ilevk**
k-Fold CV
  - Add k-Fold Cross validation
  - Add getDataLoader(), return train&valid dataloader with index list

wandb
  - Logging configs ex) batch_size, epochs, lr
  - Logging train & valid phase loss&metric

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -11,6 +11,9 @@
 from model import *
 from loss import create_criterion
 
+from sklearn.model_selection import StratifiedKFold
+
+import wandb
```
Comment (MoMentum99) [inline train.py:16]: scikit-learn 이랑 wandb 도 `requirements` 에 넣어주시면 좋을거 같습니당

File: train.py [hunk 2]
```diff
@@ -39,122 +65,129 @@ def seed_everything(seed):
...
+
+        # -- loss & metric
+        criterion = create_criterion(criterion_name)
+        optimizer = Adam(filter(lambda p: p.requires_grad, model.parameters()), lr=lr, weight_decay=5e-4)
+        scheduler = StepLR(optimizer, lr_decay_step, gamma=0.5)
+        # metrics = []
+        # callbacks = []
+
+        # -- logging
+        logger = SummaryWriter(log_dir=f"results/cv{i}_{name}")
```
Comment (MoMentum99) [inline train.py:108]: wandb 로 로깅하면 텐서보드는 없앨까요?

**Review by MoMentum99 — APPROVED** [review]
LGTM

**Comment by MoMentum99** [comment]
devel 브랜치는 학생들한테 처음 주는 코드 느낌으로 가져가면 좋을거 같아서 요거는 kfold 같은 이름으로 브랜치 분리해도 좋을거 같습니당!

**Comment by Ilevk** [comment]
브랜치 변경해서 pr 다시 올리겠습니당
